### PR TITLE
Gravatar

### PR DESCRIPTION
**Pull request by nschloe**
Check out default alternatives for Gravatar at <https://en.gravatar.com/site/implement/images/>.

### CODE REVIEW

**Comment by andrenarchy** [comment]
I've no strong feelings about the fallback gravatar pics. I like both the *identicon* and *retro* styles.

The Gravatar URLs are a nice occasion for writing an [angular filter function](https://docs.angularjs.org/guide/filter).

Pros:
* it suffices to write `{{author.gravaterMd5 | gravatarUrl:30}}` in a template (where 30 is the size parameter)
* the Gravatar URL generation is not repeated throughout the code

Cons:
* none ;)

@nschloe: do you want to give it a shot for this PR?

**Comment by nschloe** [comment]
Good idea; I'll take that on tomorrow.

**Comment by andrenarchy** [comment]
Yeah, let's call it a day! :smiley:

File: src/js/directives/gravatar.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            element.attr('width', scope.gravatarSize + 'px');
+            element.attr('height', scope.gravatarSize + 'px');
+            element.attr(
+              'src',
+              "https://secure.gravatar.com/avatar/" +
+              user.gravatarMd5 +
+              "?s=" + scope.gravatarSize +
+              "&d=identicon"
+            );
+            element.attr('alt', "\"" + user.displayName + " avatar\"");
```
Comment (andrenarchy) [inline src/js/directives/gravatar.js]: `\"` should be removed (both)

**Comment by andrenarchy** [comment]
Nice, the code looks so much cleaner with the directive! :+1:

**Pull request merged**